tests/bson/bson-generate-document-id.phpt

```diff
@@ -48,23 +48,23 @@ array(3) {
     string(8) "New York"
   }
   ["_id"]=>
-  object(%s\ObjectID)#%d (%d) {
+  object(%s\ObjectID)#%d (1) {
     ["oid"]=>
     string(24) "%s"
   }
 }
 
 Dumping fetched user document:
-array(3) {
+object(stdClass)#%d (3) {
   ["_id"]=>
-  object(%s\ObjectID)#%d (%d) {
+  object(%s\ObjectID)#%d (1) {
     ["oid"]=>
```

Comment:
I didn't see the harm in hard-coding these because the elements themselves are rigidly defined below. If more elements were added, we'd have to update both anyway.

I can make a commit changing this across the board, though. In the long run, it would reduce the size of diffs when we add elements down the line and take away one more thing that might get out of sync.